### ISSUE TITLE
Emblems of Heroism for gloves and chest of lost conqueror

### ISSUE THREAD

**Issue by bctaylor1** [dalaranwow/dalaran-wow#5526]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Bought the pieces of gear with the emblems of heroism. Unable to create the actual piece of armor for my paladin.
**Expected behaviour**: Tell us what should happen instead
Right click and create a piece of wearable armor. Tried on multiple days and got the same results
**Steps to reproduce the problem**:

1.  bought item token
2. right click token 
3. No ablility to create actual armor

**Include proofs for this behaviour**
![image](https://user-images.githubusercontent.com/40504669/41801060-565a6f66-7646-11e8-9a68-84386c922048.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by mrnuggets** [comment]
Sorry for the cancerous censor.

I think you got it all wrong, you are supposed to go to this vendor in Dalaran, and buy the set pieces.

https://imgur.com/apTr5PP

She's located at the blacksmithing, close to the "horde" bank.

It functions as it should.
